src/glmfit.jl

```diff
@@ -89,6 +134,23 @@ function wrkresp(r::GlmResp)
     isempty(r.offset) ? tmp : broadcast!(-, tmp, tmp, r.offset)
 end
 
+"""
+    wrkresp!{T<:FPVector}(v::T, r::GlmResp{T})
+
+Overwrite `v` with the working response of `r`
+"""
+function wrkresp{T<:FPVector}(v::T, r::GlmResp{T})
+    broadcast!(+, v, r.eta, r.wrkresid)
+    isempty(r.offset) ? v : broadcast!(-, v, v, r.offset)
+end
+
+"""
+    wrkwt(r::GlmResp)
+
+Return the working weights.
+
+The result is `r.wrkwt`, not a copy of `r.wrkwt`
```

Comment:
Also it's a bit weird to say this is `r.wrkwt` in the docs: if that's guaranteed to be the case, why provide an accessor at all?